src/main/asciidoc/inc/start/_configuration.adoc

```diff
@@ -9,6 +9,10 @@ In addition to the <<global-configuration>>, this goal supports the following gl
 | Default pattern for naming all containers when they are created. See <<container-name, Container Names>> for details.
 | `docker.containerNamePattern`
 
+| *platform*
```

Comment:
@chonton This documentation is in the wrong place, it should be under the `<run>` section below.